client/src/menu/main/StartMenu.tsx

```diff
@@ -18,8 +19,8 @@ export default class StartMenu extends React.Component<StartMenuProps, StartMenu
 
     private async connectButton() {
         Anchor.setContent(<LoadingScreen type="default"/>);
-
-        GAME_MANAGER.setOutsideLobbyState();
+        await GAME_MANAGER.setOutsideLobbyState();
+        Anchor.setContent(<PlayMenu/>);
```

Comment:
These can be inside the function again

Reply:
Nope, in index.tsx when auto-reconnecting is handled, we want this to happen in the background without the user seeing the play menu and disappear for a fraction of a second